wheels/build/wheels.yml

```diff
@@ -1,6 +1,10 @@
 ---
 
 packages:
+    biopython:
+        version: 1.66
+        prebuild:
+            wheel: "git clone -b linux-wheels https://github.com/natefoo/pip && cd pip && for py in /python/cp*-`uname -m`; do ${py}/bin/python setup.py install && ${py}/bin/pip install --index-url https://wheels.galaxyproject.org/simple/ numpy || exit 1; done"
```

Comment:
Sorry for the bad info @bgruening, this is outdated for manylinux1 wheels. You should not need to install Galaxy pip, so this line can become:

```yaml
wheel: "for py in /python/cp*-`uname -m`; do ${py}/bin/pip install --index-url https://wheels.galaxyproject.org/simple/ numpy || exit 1; done"
```